src/frontend/actions/user-actions/user-actions.ts

```diff
@@ -87,13 +87,13 @@ export class UserActions {
    * Update the node state after re rendering the tree.
    * Select the previously selected node and
    * Preserve state of previously Closed Component.
-   * @param  {Object} options.openedNodes list of opened Nodes id's
+   * @param  {Object} options.closedNodes list of opened Nodes id's
```

Comment:
@vanessayuenn I am guessing you also want me to change `opened Nodes` to `closed Nodes` in the comment doc string. I just want to be absolutely certain.